packages/app-store/whatsapp/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "/*": "Don't modify slug - If required, do it using cli edit command",
+  "name": "WhatsApp",
+  "slug": "whatsapp",
+  "type": "whatsapp_video",
+  "imageSrc": "/api/app-store/whatsapp/icon.svg",
+  "logo": "/api/app-store/whatsapp/icon.svg",
+  "url": "https://cal.com/apps/whatsapp",
+  "variant": "conferencing",
+  "categories": ["video"],
+  "publisher": "Cal.com, Inc.",
+  "email": "support@cal.com",
+  "description": "Schedule a chat with your guests or have a WhatsApp Video call.",
+  "extendsFeature": "User",
+  "__createdUsingCli": true,
+  "appData": {
+    "location": {
+      "type": "integrations:whatsapp_video",
+      "label": "WhatsApp",
+      "linkType": "static",
+      "organizerInputPlaceholder": "https://wa.me/send?phone=491512233334",
```

Comment:
Is this a real phone as a placeholder? perhaps it would be better to invent one like 1234567890 or XXXXXXXX

Reply:
its not a real number but it could become one. good point. will do 12345